docs/docker_quickstart.md

```diff
@@ -22,10 +22,10 @@ Freqtrade provides an official Docker image on [Dockerhub](https://hub.docker.co
 
 ### Docker quick start
 
-Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
+Create a new directory and place the [docker-compose file](https://raw.githubusercontent.com/freqtrade/freqtrade/stable/docker-compose.yml) in this directory.
 
-=== "PC/MAC/Linux"
-    ``` bash
+#### PC/MAC/Linux
```

Comment:
This will break / change rendering on freqtrade.io (the same will happen with all other below changes which replace `!!!` with `###`).

I understand why you're doing it (regular markdown programs will not recognize this format) - but the main purpose of the docs folder is the rendering on freqtrade.io.

you can look at the [developer documentation](https://www.freqtrade.io/en/latest/developer/#documentation) on how to serve the documentation locally to check if it renders correctly.